applications/luci-app-usteer/root/etc/uci-defaults/80_usteer.txt

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
An empty/no-op uci-defaults file should be removed.

Reply:
ok will do